server/db/driver/pg/internal/matches.go

```diff
@@ -150,4 +150,10 @@ const (
 
 	SetSwapDone = `UPDATE %s SET active = FALSE
 		WHERE matchid = $1;`
+
+	SelectMatchStatuses = `SELECT matchid, status, aContract, bContract, aContractCoinID,
+		bContractCoinID, aRedeemCoinID, bRedeemCoinID, aRedeemSecret
+		FROM %s
+		WHERE matchid = ANY($2)
+		AND (takerAccount = $1 OR makerAccount = $1);`
```

Comment:
I'd expect the `active` column would be needed here to infer revocation.  And for the route, we'd want to communicate that to the user too.  I don't think I thought that through in the spec PR, but status only says when the match was revoked (if it was revoked), so it seems like we need another field to make the route useful to the client.

Reply:
Right! Match status does not say `Revoked` for revoked matches.